### ISSUE TITLE
Saroo wont play backup cd's?

### ISSUE THREAD

**Issue by Junijam922** [tpunix/SAROO#156]
Tried running a backup cd and wont load disc.  Error comes up with "NO DISC FOUND!".   Does Saroo only play authentic Saturn disc games not backups?

**Comment by ragingchaos** [comment]
Booting CDRs with Saroo is currently not supported.

**Comment by Tissemon** [comment]
Is there any plans on adding support for CDR-games? If I run into a game that won't work with Saroo, I would like the option to just burn that game to a CD and play it that way instead. I've got an AR with Pseudo Saturn Kai Lite on it, so I can technically use that instead. But now that I've got a Sara, I never want to take it out of my Saturn :-)

**Comment by Junijam922** [comment]
Need to add modchip...then cdr will work with Saroo no issues.

**Comment by Tissemon** [comment]
Yes. But as I wrote: I can already play backups with Pseudo Saturn Kai. And the source code for Pseudo Saturn is available here: https://github.com/cyberwarriorx/pseudosaturn so it shouldn’t be too hard to implement that code in to Saroo. So is there a special reason why backups can’t be loaded with Saroo?

**Comment by yzbss** [comment]
For no reason, it will be added later
